src/background/popupHandler.ts

```diff
@@ -17,7 +17,7 @@ export const getAeppUrl = (v: any) => new URL(v.connection.port.sender.url);
 export const showPopup = async (aepp: any, type: string, params?: any) => {
   const id = uuid();
   const { href, protocol, host } = typeof aepp === 'object' ? getAeppUrl(aepp) : new URL(aepp);
-  const tabs = await browser.tabs.query({ active: true });
+  const tabs: any[] = await browser.tabs.query({ active: true });
```

Comment:
```suggestion
interface Tabs {
  url: string,
  tabId: number
}
  const tabs = await browser.tabs.query({ active: true }) as Tabs[];
```

Reply:
Both options: `any` or the one suggested by Konrad makes no sense in my opinion because on line 11 we define the type of the `browser` as the `Browser` (taken from the polyfill). So without any additional typing the `tabs` const has correct type:
![image](https://github.com/aeternity/superhero-wallet/assets/8217670/37753868-6d7c-4d74-a3ce-26e4d0d6ad21)